Things_You_Should_Know/things_you_should_know.md

```diff
@@ -0,0 +1,400 @@
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ✅    | ✅          |
+
+We offer a **BASH** console in [Kudu (Advanced Tools)](https://github.com/projectkudu/kudu/wiki) that you can use to browse directory structure, etc. It's important to realize that this console is running inside of **Kudu**. This is not the same container that your app is running in, so if you look at files and directories here, they won't reflect your app. The only exception to that is the /home directory. That directory is, by default, mounted from **App Service** built in storage, and it's the same location in both the **Kudu** and your app's container. [Learn more about persistent storage](#You-can-enable-and-disable-storage-persistence-with-an-app-setting)
+
+### You don't need (and should not have) SSL/TLS support in your app
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ✅    | ✅          |
+
+**App Service** terminates SSL/TLS at the front-ends. That means that SSL/TLS requests never get to your app. That's good news for you because it means that you don't need to (and should not) implement any support for SSL/TLS into your app. Also as stated above, it's important to understand that the front-ends where SSL/TLS is terminated are inside of our Azure data centers. If you use SSL/TLS with your app, your traffic across the Internet will always be safely encrypted.
+
+### You can only expose one port to the outside world
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ⛔    | ✅          |
+
+**Container** apps allow you to expose only one port to the outside world. That means that your container can only listen for HTTP/HTTPS requests on a single port. Some apps need multiple ports. For example, you might have one port that is used for requests into the app and a separate port that is used for a dashboard or admin portal. As of today, that configuration isn't possible in **App Service**.
+
+We will attempt to detect which port to bind to your container, but you can also use the `WEBSITES_PORT` app setting and configure it with a value for the port you want to bind to your container.
+
+> **Note**
+>
+> **Bring your own Code** apps use port `80` and `443` by default and can't be configured to use alternate ports.
+
+### Don't use df to find your disk space usage
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ✅    | ✅          |
+
+You're probably used to using `df` to find out disk space in your volumes. You can certainly do that in **App Service**, but what you'll see isn't really going to be as helpful as you might think.
+
+Assuming you are persisting files in the /home directory (which is currently the default), you will have a quota based on your pricing tier. However, if you use `df`, the disk space you see **won't reflect that quota**. If you want to find out your disk usage, use **Quotas** in your app's menu, **File System Storage** in the **App Service Plan** menu, or use the REST API to find your usage.
```

Comment:
This is a super important point!  Customers using df will think there's a petabyte of space, when in reality they only have 10GB or less.  Further, there's no way to find out what the quota is, or how close they are to the limit.